packages/form-js-editor/src/render/hooks/useDebounce.js

```diff
@@ -0,0 +1,25 @@
+import {
+  useMemo,
+  useEffect
+} from 'preact/hooks';
+
+import useService from './useService';
+
+
+export default function useDebounce(fn, dependencies=[]) {
+
+  const debounce = useService('debounce');
+
+  const callback = useMemo(() => {
+    return debounce(fn);
+  }, [...dependencies, fn ]);
```

Comment:
```suggestion
  }, dependencies);
```

Reply:
Otherwise it's flushed on each render if `fn` is defined within the render function.